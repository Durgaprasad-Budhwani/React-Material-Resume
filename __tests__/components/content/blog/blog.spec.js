/**
 * Created by Durgaprasad Budhwani on 10/9/2017.
 */
/* global expect,it */
import React from 'react';
import BlogContainer, {Blog} from '../../../../src/components/content/blog/blog';
import renderer from 'react-test-renderer';

it('renders <BlogContainer />', () => {
    const tree = renderer.create(
        <BlogContainer
            md={6}
            sm={12}
            image={'image1'}
            title={'title1'}
            day={'day1'}
            month={'month1'}
            url={'url'}
        />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});


it('renders <Blog />', () => {
    const classes = {
        blogItemImage: {},
        blogItemBox: {},
        imageContainer: {},
        blogItemContent: {},
        blogItemTimeContainer: {},
        blogItemTimeDay: {},
        blogItemTimeMonth: {},
        blogItemTitle: {}
    };
    const tree = renderer.create(
        <Blog
            md={6}
            sm={12}
            image={'image1'}
            title={'title1'}
            day={'day1'}
            month={'month1'}
            url={'url'}
            classes={classes}
        />
    );
    const instance = tree.getInstance();
    instance._openTab('url');
    expect(tree.toJSON()).toMatchSnapshot();
});

