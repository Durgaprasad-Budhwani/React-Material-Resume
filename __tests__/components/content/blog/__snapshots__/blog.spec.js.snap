// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders bubble 1`] = `
<div
  className="MuiGrid-typeContainer-9 MuiGrid-typeItem-10 MuiGrid-spacing-xs-16-24 MuiGrid-direction-xs-column-11 MuiGrid-justify-xs-center-19 MuiGrid-grid-sm-12-52 MuiGrid-grid-md-6-59"
>
  <div
    className="MuiPaper-root-92 MuiPaper-shadow2-96 MuiPaper-rounded-93 Blog-blogItemBox-2"
  >
    <div
      className="Blog-imageContainer-3"
    >
      <img
        alt="title1"
        className="Blog-blogItemImage-1"
        src="image1"
      />
    </div>
    <div
      className="Blog-blogItemContent-4"
    >
      <time
        className="Blog-blogItemTimeContainer-5"
      >
        <div
          class="Blog-blogItemTimeDay-6"
        >
          day1
        </div>
        <div
          class="Blog-blogItemTimeMonth-7"
        >
          month1
        </div>
      </time>
      <h3
        className="Blog-blogItemTitle-8"
      >
        title1
      </h3>
    </div>
  </div>
</div>
`;
